cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1589,7 +1486,7 @@ SwiftGenerator::writeMarshalUnmarshalCode(
                     {
                         args += ", type: ";
                     }
-                    args += getUnqualified(getAbsolute(type), swiftModule) + ".self";
+                    args += getUnqualified("Ice.ObjectPrx", swiftModule) + ".self";
```

Comment:
For another PR. We should cleanup the code to not use getUnqualified for Ice module, like we did in other languages.